ngx-tools/testing/src/mocks/token-escalator.mock.ts

```diff
@@ -91,5 +93,6 @@ export class TokenEscalatorMock<CM = ClaimMap> implements TokenEscalator<CM> {
 }
 
 export function tokenEscalatorFactory() {
+  // tslint:disable-next-line deprecation
```

Comment:
JSDoc comments about deprecation go on the item that is deprecated. This is simply an internal comment for our own linter because we are using a deprecated item.